docs/signing-spec.md

```diff
@@ -217,13 +190,51 @@ Verification of an invoice includes the following steps:
 3. Reconstruct the cleartext block following the signing rules
 4. For each signature block
     a. Extract the public key
-    b. Verify the signature using the public key and the cleartext
+    b. Verify that the key has not already been used in another signature block
+        - If a key is used to sign the same invoice multiple times, the implementation SHOULD fail
+    c. Verify the signature using the public key and the cleartext d
```

Comment:
There is a trailing `d` here that should be removed

Reply:
Fixed